src/index.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+//import { Router, Route, hashHistory } from 'react-router'
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+
+import App from './App';
+import LoginPage from './LoginPage';
+
+import registerServiceWorker from './registerServiceWorker';
+import './index.css';
+
+/*
+ReactDOM.render(
```

Comment:
You can delete these comments.

Reply:
Done in the branch Mathieu/feat/LoginPage